ci-operator/step-registry/openshift/upgrade/gcp/ovn/openshift-upgrade-gcp-ovn-workflow.metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+	"path": "openshift/upgrade/gcp/ovn/openshift-upgrade-gcp-ovn-workflow.yaml",
+	"owners": {
+		"approvers": [
+			"danwinship",
+			"dcbw",
+			"knobunc",
+			"squeed"
+		],
+		"reviewers": [
+			"alexanderconstantinescu",
```

Comment:
s/alexanderconstantinescu/alexanderConstantinescu/g

also

s/jacobtanenbaum/JacobTanenbaum/g

here and in `OWNERS`

though given how little I know about CI orchestration / configuration, I am not sure I should be a reviewer...

Reply:
Teams are responsible for maintaining their own CI bits, so yes, everyone should be here.

This is a pretty old list though. It should be synced to https://github.com/openshift/ovn-kubernetes/blob/master/OWNERS though